lib/package/bundleLinter.js

```diff
@@ -179,7 +179,7 @@ var getFormatter = function(format) {
 var executePlugin = function(file, bundle, callback) {
   if (file.endsWith(".js")) {
     var plugin = require("./plugins/" + file);
-    if (plugin.plugin.enabled) {
+    if (plugin.plugin.enabled && !excluded.included(plugin.ruleId)) {
```

Comment:
```suggestion
    if (plugin.plugin.enabled && !excluded.includes(plugin.plugin.ruleId) ) {
```

Reply:
I updated this check to better reflect the data structure.